src/Agent/AgentState.vue

```diff
@@ -75,10 +75,10 @@
       <div class="col-5 agent-state-text">Activity Time:</div>
       <div class="col-7 agent-state-text"> {{msToHms(this.time_activity) }} </div>
     </div>
-    <div v-access:myPhone-feature class="row">
+    <div v-if="$agent.permAllowed('myPhone-feature')" class="row">
```

Comment:
IMO v-access:myPhone-feature way more readable than $agent.permAllowed('myPhone-feature'). Why to change?

Reply:
Because you can easily remove "display :none" option through developer tools of any browsers and allow my-phone feature. Its not possible in case of v-if condition.

Reply:
I'm quite sure you can change access directive to do the same.

Reply:
I do not see any reason to use custom directive to do the same things as system directive

Reply:
Directives are used to abstract things and to make the intent clear.